### DISCUSSION TITLE
Exporting burn model.

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#2431] [Q&A]
Hi,
How to export burn models so that we can use it in other frameworks such as pytorch ?

**Comment by HessTaha** [comment]
Hello,

I believe you can export the model in ONNX from there you Can load it back using pytorch? 

PS: I've never tried that

**Comment by laggui** [comment]
We support ONNX import but not export.

There is no official way to export to another format, but if you save it with one of the json file recorders you could load the weights in python with your framework of choice.

For pytorch you'll probably need to map the key-value pairs to the correct state_dict based on your model definition.

P.S: questions of this nature are usually better suited for [discussions](https://github.com/tracel-ai/burn/discussions) 🙂

**Comment by antimora** [comment]
The default file format is in [message pack](https://msgpack.org/index.html) which is portable and supported by many languages. JSON also works but it could be large.
